package.json

```diff
@@ -28,7 +28,7 @@
   },
   "homepage": "https://github.com/rdfjs-base/data-model",
   "dependencies": {
-    "@types/rdf-js": "*"
+    "@rdfjs/types": "*"
```

Comment:
This is a nit, but was there a moment in time where `@types/rdf-js` types were moved to this package? So wouldn't it make sense to pin it to that version?

Reply:
It's not a great idea pin types packages because it can cause duplicate resolution and false-negative build errors

Reply:
I was not so much talking about pinning, but using a minimum, equivalent to the first published, probably (or the second one)

Reply:
Sorry, I phrased this wrong. What I meant is that packages from "incompatible" versions of referenced `@types/rdf-js` will inevitably result in duplicate resolutions. Namely, depending on different major versions

Reply:
we could specify `"@rdfjs/types": ">=1.0.1"`, which a) uses a minimum version to show when this switched over, while b) not specifying only 'compatible' versions (as with `"@rdfjs/types": "^1.0.1"`), so hopefully avoiding the duplicate resolution issue?

Basically, should be functionally equivalent to using `*` but with a _bit_ more info?

Reply:
Ah, this should work. Somehow I didn't realise that `>` ranges are also allowed in dependencies.